website/content/en/about/corporate-information.md

```diff
@@ -16,7 +16,7 @@ GERMANY
 
 ### Members of the Board of Management
 
-Dr. Volkmar Denner, Prof. Dr. Stefan Asenkerschbaumer, Dr. Michael Bolle, Dr. Christian Fischer, Dr. Stefan Hartung, Dr. Markus Heyn, Harald Kröger, Christoph Kübel, Rolf Najork, Uwe Raschke, Peter Tyroller
+Dr. Volkmar Denner, Prof. Dr. Stefan Asenkerschbaumer, Dr. Michael Bolle, Dr. Christian Fischer, Dr. Stefan Hartung, Dr. Markus Heyn, Harald Kröger, Filiz Albrecht, Rolf Najork, Uwe Raschke
```

Comment:
Except for the CEO, the current list appears to be sorted alphabetically...
```suggestion
Dr. Volkmar Denner, Filiz Albrecht, Prof. Dr. Stefan Asenkerschbaumer, Dr. Michael Bolle, Dr. Christian Fischer, Dr. Stefan Hartung, Dr. Markus Heyn, Harald Kröger, Rolf Najork, Uwe Raschke
```

Reply:
@choeppler thanks for the observation.